src/components/Index.tsx

```diff
@@ -0,0 +1,19 @@
+import { Routes, Route } from "react-router-dom";
+import Home from "./Home";
+
+import MovieDetail from "./common/MovieDetail";
+import Login from "./Login";
+import Registration from "./Registration";
+
+export default function Index() {
+	return (
+		<div>
+			<Routes>
+				<Route path="/" element={<Home />} />
+				<Route path="login" element={<Login />} />
+				<Route path="register" element={<Registration />} />
+				<Route path="detail/:id" element={<MovieDetail />} />
+			</Routes>
```

Comment:
put all the routes in different file and that file name should be routes or routing  something like this